apps/kg/kvclient.py

```diff
@@ -191,7 +166,8 @@ def get_local_machine_id(server_namebook):
 def start_worker(args, logger):
     """Start kvclient for training
     """
-    train_time_start = time.time()
+    init_time_start = time.time()
+    time.sleep(5) # wait for launch script
```

Comment:
is it always successful?

Reply:
I change it to WAIT_TIME, which is defined as 10 seconds. I tested it for ssh 200 nodes and it is no problem.